src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/FullClr/WebRequestPSCmdlet.FullClr.cs

```diff
@@ -84,10 +84,22 @@ internal virtual WebRequest GetRequest(Uri uri)
                 request.Proxy = WebSession.Proxy;
             }
 
-            // set the method if the parameter was provided
-            if (WebRequestMethod.Default != Method)
+            switch (ParameterSetName)
```

Comment:
I had this simplified to this. Since CustomMethod will always be null or a real string (param validation) checking for $null should be enough.

```csharp
// set the custom method if the parameter was provided
            if(CustomMethod != null)
            {
                request.Method = CustomMethod.ToUpperInvariant();
            }
            else
            {
                // set the method if the parameter was provided
                if (WebRequestMethod.Default != Method)
                {
                    request.Method = Method.ToString().ToUpperInvariant();
                }
            }
```